tsconfig.json

```diff
@@ -1,37 +1,44 @@
 {
   "compilerOptions": {
-    "target": "es2018",
-    "lib": [
-      "dom",
-      "esnext"
-    ],
-    "outDir": "dist",
-    "declaration": true,
-    "sourceMap": true,
+    "allowJs": true,
+    "target": "esnext",
```

Comment:
that could\will cause problems at least in test. there problem with optional chaining and esnext. thats why it was es2018